frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html

```diff
@@ -7,7 +7,7 @@
             type="submit"
             mat-flat-button
             class="new-meeting-header-button"
-            [disabled]="meetingForm.invalid || !meetingForm.touched">
+            [disabled]="meetingForm.invalid || !meetingForm.touched || addedMembers.length === 0">
```

Comment:
```suggestion
            [disabled]="meetingForm.invalid || !meetingForm.touched || !addedMembers.length">
```

Reply:
Add logic to show current user as team member on creating meeting.